plaso/parsers/firefox_cache.py

```diff
@@ -135,9 +135,7 @@ class FirefoxCacheParser(BaseFirefoxCacheParser):
   """Parses Firefox cache version 1 files (Firefox 31 or earlier)."""
 
   NAME = 'firefox_cache'
-  DESCRIPTION = (
-      'Parser for Mozilla Firefox Cache version 1 files (version 31 or '
-      'earlier).')
+  DATA_FORMAT = 'Mozilla Firefox Cache version 1 file (version 31 or earlier)'
```

Comment:
I don't see the corresponding change in utils/export_supported_formats.py, is this normal?

Reply:
Good catch, I've made the additional changes. FYI I'll do a pass over export_supported_formats after making all the changes to the parses.